storage/innobase/log/log0log.cc

```diff
@@ -62,6 +62,8 @@ Created 12/9/1995 Heikki Tuuri
 
 #include "system_key.h"
 
+static redo_log_encrypt_enum found_log_encryption_mode = REDO_LOG_ENCRYPT_OFF;
```

Comment:
Can we reuse srv_redo_log_encrypt ? By doing so, we show the correct log encryption typed used? i.e with SELECT @@innodb_redo_log_encrypt?

Some question related:
Looks like the variable is dynamic. Can we just change this variable from RK to MK and MK to RK ?

Reply:
We can't: this variable stores what encryption type we found in the existing redo log, even if it is currently turned off.
This is because if it used master_key in the past (has encrypted data / encryption key in it), we have to rotate it during rotate innodb master key, even if the setting is currently off.
Without our redo encryption changes, this was trivial (if it was encrypted previously, it was always master key encrypted), but now we added the keyring_key option, which means we have to store the encryption mode somewhere.